modules/series-service-impl/src/main/java/org/opencastproject/series/endpoint/SeriesRestService.java

```diff
@@ -336,10 +336,10 @@ public Response addOrUpdateSeries(@FormParam("series") String series, @FormParam
     try {
       dc = this.dcService.load(new ByteArrayInputStream(series.getBytes("UTF-8")));
     } catch (UnsupportedEncodingException e1) {
-      logger.error("Could not deserialize dublin core catalog: {}", e1);
+      logger.error("Could not deserialize dublin core catalog: {}", e1.getMessage(), e1);
```

Comment:
Think it's simpler and equivalent just to do:

      logger.error("Could not deserialize dublin core catalog", e1);

Reply:
It looked as though the logger.error() wasn't picking up the throwable in the message, but I'll revert that and remove the unused imports. Thanks!

Reply:
I mean the original is wrong in some way: either the intention is to log only the message in which case it should have been:

      logger.error("Could not deserialize dublin core catalog: {}", e1.getMessage());

or the intention was to log a full stack trace in which case it should have been:

      logger.error("Could not deserialize dublin core catalog", e1);

Reply:
For an error, please use the latter. An error indicated that something is actually wrong. Hence, having a stack trace is really helpful.